dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/UniqueFieldValidationStrategy.java

```diff
@@ -120,4 +121,46 @@ default void validateField(final Field field) {
         }
     }
 
+    /**
+     * Clean the Extra unique validation field table after a {@link Contentlet} have been removed.
+     * We need to remove all the unique values of this {@link Contentlet} and {@link com.dotmarketing.portlets.languagesmanager.model.Language}
+     * from the extra table.
+     *
+     * @param contentlet
+     */
+    default void cleanUp(final Contentlet contentlet, final boolean deleteAllVariant) throws DotDataException {
+        //Default implementation do nothing
+    }
+
+    /**
+     * Method call after publish a {@link Contentlet} it allow thw {@link UniqueFieldValidationStrategy} do any extra
```

Comment:
the* (please read what's written)

Reply:
done https://github.com/dotCMS/core/pull/30798/commits/44e8e93dd827f27595fe6f8674e3ba6d598ab341#diff-445f8d01aa4de058eaaf883e573d17ef1123b1e74a9a98120ac156b78f4c6522R135